s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3Client.scala

```diff
@@ -56,9 +56,14 @@ final class S3Client(s3Settings: S3Settings, system: ActorSystem, mat: Materiali
     impl.download(S3Location(bucket, key), Some(scalaRange)).asJava
   }
 
-  // #list-bucket
```

Comment:
Maybe they should? Im not quite sure. It seems like they should. Would it be best to remove the scaladoc I put in and just the paradox #list-bucket?